roles/ceph-osd/tasks/activate_osds.yml

```diff
@@ -3,7 +3,7 @@
 # partition.
 
 - name: automatically activate osd disk(s) without partitions
-  command: ceph-disk activate "/dev/{{ item.key }}"
+  command: ceph-disk activate "/dev/{{ item.key }}p1"
```

Comment:
this is only valid for NVMe drives I suppose, not generic enough and this will break current and new deployments.